src/content/protocols/pancakeswap-v2-bsc.md

```diff
@@ -0,0 +1,150 @@
+
+The project could advance to Stage 2 if every critical permission concerning CAKE fee incentive distribution was irrevocably assigned to on-chain governance and protected by a long Exit Window, with multiple independent front-ends ensuring continuous user access.
+
+# Reviewer's Notes
+
+PancakeSwap follows the standard Uniswap V2 pattern with the addition of a incentive layer (V1 + V2). The way they upgraded from PancakeSwap V1 to V2 the MigrationHelper was not in the scope of this review.
+
+# Protocol Analysis
+
+https://app.excalidraw.com/l/9pt8PDVB43r/k5Y6Daznbm
+
+PancakeFactory : Acts as the on‑chain directory and deployer for every liquidity pool. When createPair(tokenA, tokenB) is called it deterministically clones the PancakePair template and initialises the new pair. Two mutable variables govern fees: feeTo (recipient of the 0.08 % protocol cut) and feeToSetter (address allowed to change feeTo). Both can be changed with a 3‑of‑6 multisig call, making Factory the choke‑point for all AMM fee flows.
+
+PancakeRouter V2 : A stateless helper that strings together common user actions. During swaps it calculates the best path through one or more PancakePair contracts, pulls the caller’s tokens, and routes them hop‑by‑hop, enforcing the user’s amountOutMin and deadline. For liquidity it wraps addLiquidity/removeLiquidity, handles slippage limits and token approvals, and wraps or unwraps WBNB when native BNB is involved. After execution it holds no residual balances.
+
+MasterChef V1 : The legacy farming contract that mints a fixed 40 CAKE every block. Ninety percent goes to pool rewards and ten percent to a dev address that the multisig can switch instantly via dev() (no Timelock). Because all CAKE supply originates here, V1 remains the root monetary‑policy lever even after V2 launched.
+
+MasterChef V2 : The active farming layer where users stake LP tokens to earn CAKE. It receives freshly‑minted CAKE from V1 and allocates it to pools based on allocPoint. Governance (Timelock‑gated multisig) can add/remove pools, adjust weights, and tweak the burn/reward split.
+
+# Dependencies
+
+This protocol is deployed on **BNB Smart Chain**; if the chain halts, censors transactions, or suffers prolonged congestion, every swap, liquidity movement, and MasterChef call is blocked. In such an outage the protocol’s pools become illiquid and the emission schedule pauses because new CAKE cannot be minted on a stalled L1.
+
+All mutable parameters pass through two **Gnosis Safe 3-of-6 multisigs**. If a quorum cannot be reached—because keys are lost, signers go offline, or signatures are withheld—routine updates such as adding a new farm or adjusting emission rates become impossible. Conversely, if an attacker compromises three keys, they can immediately redirect the 0.08 % protocol fee or the full CAKE emission stream; the optional timelock offers no protection against direct multisig calls in MasterChef V1.
```

Comment:
Those aren't dependencies. Here you explain the technical details of what is mentioned in the "Autonomy" section